### ISSUE TITLE
questions in  pretrained log by upernet_deit-b16_512x512_160k_ade20k.py

### ISSUE THREAD

**Issue by haoxurt** [open-mmlab/mmsegmentation#1606]
I wonder how many gpus used when you trained by upernet_deit-b16_512x512_160k_ade20k.py .   In the provided log  [https://download.openmmlab.com/mmsegmentation/v0.5/vit/upernet_deit-b16_512x512_160k_ade20k/20210621_180100.log.json](url), one epoch include 1250 iterations,so the batch is 16.  However, the gpu_ids is in range(0, 1), samples_per_gpu is 2. In my opinion, the gpu_ids should be  in range(0, 8).  What's more, I also wonder  the learning rate.

**Comment by linfangjian01** [comment]
lr in  './upernet_vit-b16_mln_512x512_160k_ade20k.py', and total batch size = 16. We use 8 gpus for training, you can find it in the log, searching for the keyword 'ngpus'.

**Comment by haoxurt** [comment]
Thanks for replying. However, I can't find the keyword 'ngpus' in the log [https://download.openmmlab.com/mmsegmentation/v0.5/vit/upernet_deit-b16_512x512_160k_ade20k/20210621_180100.log.json](url). I only  finded keyword gpu_ids = range(0, 1).    I also find description about lr in [https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/train.md](url), 'The default learning rate in config files is for 4 GPUs and 2 img/gpu (batch size = 4x2 = 8)'.   In the log, lr is 0.00006.  0.00006 is corresponding to batch size 16?

**Comment by linfangjian01** [comment]
In https://download.openmmlab.com/mmsegmentation/v0.5/vit/upernet_deit-b16_512x512_160k_ade20k/20210621_180100.log.json，input ‘ctrl’ + 'F',  search 'ngpu', the result is 'nGPU 0,1,2,3,4,5,6,7'.  It uses the adamw optimizer, and lr = 0.00006 is the better parameter after more experiments. batch size = 16 is the benchmark for a fair comparison of the community on the ADE20k dataset. In general the larger the batch size, the better the performance will be. I suggest you to read more about the model implemented in mmseg, and the corresponding config, and if you have time, you can also read the corresponding paper.

**Comment by linfangjian01** [comment]
My fault, the keyword should be 'ngpu'

**Comment by linfangjian01** [comment]
In https://github.com/open-mmlab/mmsegmentation/blob/master/docs/en/train.md, it is only an example file, please refer to the corresponding config file of the model.

**Comment by haoxurt** [comment]
Thanks for replying again. I will close it.
